library/src/main/java/com/getstream/sdk/chat/rest/controller/APIService.java

```diff
@@ -83,6 +84,12 @@
 
     @POST("/moderation/unflag")
     Call<FlagResponse> unFlag(@Query("api_key") String apiKey, @Query("user_id") String userId, @Query("client_id") String connectionId, @Body Map<String, String> body);
+
+    @POST("/moderation/ban")
+    Call<CompletableResponse> banUser(@Query("api_key") String apiKey, @Query("client_id") String connectionId, @Body BanUserRequest body);
```

Comment:
@bevzaanton feel free to replace other endpoints' Completable that have no data with `CompletableResponse` less code is always better :)

Reply:
Ok. I will do it my next PR